rabbitmq/publisher_test.go

```diff
@@ -69,24 +71,26 @@ func TestBroadcast(t *testing.T) {
 	require.Nil(t, err)
 
 	rabbitmq.Run()
+	defer rabbitmq.Close()
+	wg.Add(1)
 
 	rabbitmq.Broadcast(data.BlockEvents{})
 
-	<-done
+	wg.Wait()
 
-	assert.True(t, wasCalled)
+	assert.Equal(t, uint32(1), numCalls)
```

Comment:
`atomic.LoadUint32(&numCalls)` to prevent race conditions

Reply:
ahh, changed :+1: